scripts/build-manifests.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
The scripts misses license statement.  There are several shell script improvements to do, such as:

- do not use `$1` throughout the file, rather read the arguments in the beginning into variables and use those;
-  semicolons at the end of statements are not needed if there is no follow-up statement to execute
- see also `shellchech` recommendations such as `cd $somedir || exit`

Shall we do these now or via a later cleanup?